src/Utils/arm/request.ts

```diff
@@ -144,13 +144,13 @@ async function getOperationStatus(operationStatusUrl: string) {
 
   const body = await response.json();
   const status = body.status;
-  if (!status && response.status === 200) {
-    return body;
-  }
   if (status === "Canceled" || status === "Failed") {
     const errorMessage = body.error ? JSON.stringify(body.error) : "Operation could not be completed";
     const error = new Error(errorMessage);
     throw new AbortError(error);
   }
+  if (response.status === 200) {
```

Comment:
I remember there was a scenario in the control plane where it returned a 2XX status code with status that was neither canceled nor failed (was "dequeued" I believe), and we kept retrying because we did not handle that. Did we ever fix that on our end?

Reply:
I believe that is handled here. I see dequeued a lot and it is always followed up with as a success/failed/cancelled status